tests/compat.sh

```diff
@@ -97,6 +98,7 @@ print_usage() {
     printf "  -p|--peers\tWhich peers to use (Default: '%s')\n" "$PEERS"
     printf "            \tAlso available: GnuTLS (needs v3.2.15 or higher)\n"
     printf "  -M|--memcheck\tCheck memory leaks and errors.\n"
+    printf "  -C|--compatibility\tTest TLS 1.3 compatibility mode (Default: '%s')\n" "$BOXES"
     printf "  -v|--verbose\tSet verbose output.\n"
```

Comment:
Should we sort the options alphabetically? If so we might want to move the help for `-v` up.